### PR TITLE
New build system of Hass.io

### PR DESCRIPTION
**Pull request by pvizeli**
**Description:**
- [x] updates all docs
- [x] update core addons
- [x] add blogpost

### CODE REVIEW

**Comment by frenck** [comment]
I do not totally agree on the chosen path and implementation. The parent PR has been merged, ignoring my comment about having concerns.

I therefore cannot support this at this point.

**Comment by MartinHjelmare** [comment]
What are the concerns? I'm interested. Maybe link to the PR and add them there, or in a separate issue at hassio?

**Comment by frenck** [comment]
@MartinHjelmare I will provide some more explanation asap.

**Comment by frenck** [comment]
@MartinHjelmare This is the related pull request: https://github.com/home-assistant/hassio/pull/191

**Comment by frenck** [comment]
I've opened an GitHub issue in order to get a discussion started.

https://github.com/home-assistant/hassio/issues/198

**Comment by frenck** [comment]
The discussion on my concerns has been closed.
So I no longer see anything that is holding this PR back from my end.

File: source/developers/hassio/addon_config.markdown [hunk 1]
```diff
@@ -135,7 +138,9 @@ The `options` dict contains all available options and their default value. Set t
     { "username": "cheep", "password": "654321" }
   ],
   "random": ["haha", "hihi", "huhu", "hghg"],
-  "link": "http://blebla.com/"
+  "link": "http://blebla.com/",
```
Comment (frenck) [inline source/developers/hassio/addon_config.markdown]: Please use `example.com` for documentation purposes. This is the official domain reserved for this by the IANA and described in RFC 2606.

**Comment by frenck** [comment]
I've added some quick fixes to the developer documentation part.
The blog post still needs some ❤️ .

**Comment by pvizeli** [comment]
Perfect

**Comment by frenck** [comment]
Thanks for the mention in the blog post 👍
Good job guys 🥇

**Pull request merged**